libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -57,6 +57,26 @@ bool DocStringAnalyser::visit(FunctionDefinition const& _function)
 	return true;
 }
 
+bool DocStringAnalyser::visit(VariableDeclaration const& _variable)
+{
+	if (_variable.isStateVariable())
+	{
+		static set<string> const validPublicTags = set<string>{"dev", "notice", "title", "author"};
```

Comment:
Is author not only for contract?

Reply:
Also title?

Reply:
Yes, they are normally only allowed on contract. The thing is that some contract authors in the past put this "accidentally" in front of a state variable. It was ok in the past, because there was no check on state variables implemented. Looks like that they never noticed that `title` and `author` was never set. However, I added the checks here to inform the authors that this will not be allowed anymore.